src/strategies/arrow-vesting/index.ts

```diff
@@ -7,79 +7,106 @@ export const author = 'BrassLion';
 export const version = '0.1.0';
 
 const vestingFactoryAbi = [
-  "function escrows_length() public view returns (uint256)",
-  "function escrows(uint256 index) public view returns (address)",
+  'function escrows_length() public view returns (uint256)',
+  'function escrows(uint256 index) public view returns (address)'
```

Comment:
@MantisClone Make sure there are no changes in other strategies when we go here https://github.com/snapshot-labs/snapshot-strategies/pull/782/files#

Reply:
Done.